test/built-ins/Temporal/PlainDateTime/prototype/equals/normal-case.js

```diff
@@ -0,0 +1,14 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
```

Comment:
Same comment as above about the name (I suggest `basic.js`)

Reply:
done